### PR TITLE
[9.0] IMP bi_sql_editor

### PR DESCRIPTION
**Pull request by nicomacr**
IMP preview SQL expression

### CODE REVIEW

**Review by Emiq2 — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by ValentinoDefelice — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Tested on local

File: bi_sql_editor/models/bi_sql_view.py [hunk 1]
```diff
@@ -570,3 +570,9 @@ def _refresh_size(self):
                 sql_view.view_name)
             self.env.cr.execute(req)
             sql_view.size = self.env.cr.fetchone()[0]
+
+    @api.multi
+    def button_preview_sql_expression(self):
+        self.button_validate_sql_expression()
+        res = self._execute_sql_request()
+        raise UserError('\n'.join(map(lambda x: str(x), res[:100])))
```
Comment (legalsylvain) [inline bi_sql_editor/models/bi_sql_view.py:578]: Hi.
This button is raising an error, even if all is ok ? 

not sure to understand. Could you precise ? 

thanks !

Comment (nicomacr) [reply bi_sql_editor/models/bi_sql_view.py:578]: If it was done on purpose to display in the window popup the result of the SQL query.

**Review by legalsylvain — APPROVED** [review]
thanks.

**Pull request merged**